UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/Interfaces/IFileShareService.cs

```diff
@@ -4,9 +4,9 @@ namespace UKHO.MaritimeSafetyInformation.Web.Services.Interfaces
 {
     public interface IFileShareService
     {
-        Task<IResult<BatchSearchResponse>> FSSBatchSearchAsync(string searchText,string accessToken, string correlationId);
+        Task<IResult<BatchSearchResponse>> FSSBatchSearchAsync(string searchText, string accessToken, string correlationId);
         Task<IResult<BatchAttributesSearchResponse>> FSSSearchAttributeAsync(string accessToken, string correlationId);
-
         Task<Stream> FSSDownloadFileAsync(string batchId, string fileName, string accessToken, string correlationId);
+        Task<Stream> FSSDownloadZipFile(string batchId, string fileName, string accessToken, string correlationId);
```

Comment:
To be consistent with other methods:
```suggestion
        Task<Stream> FSSDownloadZipFileAsync(string batchId, string fileName, string accessToken, string correlationId);
```

Reply:
done